cns-hook

```diff
@@ -5,6 +5,11 @@ if [[ -f "${BASEDIR}/override-hook" ]]; then
     source "${BASEDIR}/override-hook"
 fi
 
-handler=$1
+# Protection for unknown functions, required by dehydrated.
+valid_passthrough_hooks=$(grep function cns-hook-util | awk '{print $2}')
```

Comment:
I think this needs to be `${BASEDIR}/cns-hook-util`, doesn't it? To make this work when you're not in the directory?